templates/_helpers.tpl

```diff
@@ -116,14 +116,14 @@
 
 {{- $_ := set .envVars "PUBLIC_REPO_REDIRECTS" "\"true\"" -}}
 {{- $_ := set .envVars "QUERY_RUNNER_URL" "http://query-runner" -}}
-{{- $_ := set .envVars "REDIS_MASTER_ENDPOINT" "redis-cache:6379" -}}
+{{- $_ := set .envVars "REDIS_CACHE_ENDPOINT" "redis-cache:6379" -}}
+{{- $_ := set .envVars "REDIS_STORE_ENDPOINT" "redis-store:6379" -}}
```

Comment:
Can we just not set these and rely on the default behavior that now exists in the code?